useful-scripts/misc-test-utils/src/utils/__tests__/guid.test.ts

```diff
@@ -0,0 +1,76 @@
+import Guid from '../Guid';
```

Comment:
The test file name also needs to be uppercased too

Reply:
Cool, didn't realize the test file had to be uppercase too.